### DISCUSSION TITLE
Overview over locally running test cases

### DISCUSSION THREAD

**Discussion by bertrandkerres** [ibpsa/project1-boptest#717] [Q&A - Software]
Hi all :wave:,

while updating my Julia package for BOPTEST v0.7.0, I ran into an issue when I wanted to started a second test locally. This doesn't seem possible with the default (i.e. `docker compose up web worker provision`). But I had lost my local objects holding the first test's id, so I didn't know how to stop it through the API.

I could reset the system through `docker compose down`/ `docker compose up ...`, but I'm thinking that many people could run into that issue now with the BOPTEST-Service running locally if they forget to stop their old test.

Some unordered thoughts:
* Is this related to the number of workers, and if so, is there a good way to increase it when running locally?
* It seems there is no good response from the API if it runs out of workers, and instead it just refuses to answer the request for a new initialization?

**Comment by bertrandkerres** [comment]
Just saw that the documentation states the way to increase the number of workers. Being able to read is a clear advantage in life :sweat_smile:

**Comment by dhblum** [comment]
Thanks @bertrandkerres for bringing this up.  Indeed, you can scale the number of workers upon deployment.  But your second point about providing a helpful response by the API to indicate there is a problem is a good one.  I just made an issue about it here: https://github.com/ibpsa/project1-boptest/issues/720.

**Reply by bertrandkerres** [reply]
Thanks @dhblum!
